apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -12,6 +12,8 @@ import {
 } from "~/libs/hooks/hooks.js";
 import { actions as userActions } from "~/modules/users/users.js";
 
+import { QuizForm } from "../../../pages/quiz/libs/components/quiz-form/quiz-form.js";
```

Comment:
```suggestion
import { QuizForm } from "~/pages/quiz/libs/components/quiz-form/quiz-form.js";
```

Reply:
I tryed this but got error 
<img width="1101" alt="image" src="https://github.com/user-attachments/assets/b79c77b9-1972-41b0-bba8-74624678cd0c">

Reply:
The path seems valid to me, try to find out what is the issue